lidar_prod/tasks/building_identification.py

```diff
@@ -48,10 +48,12 @@ def run(
         _completion_flag = self.data_format.las_dimensions.completion_non_candidate_flag
 
         log.info("Clustering of points with high building proba.")
-        pipeline = get_pipeline(input_values)
+        pipeline = get_pipeline(input_values, self.data_format.epsg)
 
         # Considered for identification:
-        non_candidates = f"({self.data_format.las_dimensions.candidate_buildings_flag} == 0)"
+        non_candidates = (
```

Comment:
same, why not only one line ?

Reply:
I think it comes from black, I'll try to re-run it on each commit to be sure everything is ok

Reply:
I think the best would be to run pre-commits on the whole repo (but in another merge request)